### ISSUE TITLE
Username is required to be the same as the Database name

### ISSUE THREAD

**Issue by JShefferPillar** [smartcitiesdata/discovery_api#279]
https://github.com/smartcitiesdata/discovery_api/blob/790542c029be0dcffaeeda87d4b715a252c2597e/rel/runtime.exs#L28

**Comment by jeffgrunewald** [comment]
Is this a feature request or a bug report?
Simple setups for postgres tend to make the database name match the username used to administer the database but this isn't a requirement from Postgres's concern. It only requires a Postgres database be created with any user account granted the necessary permissions the application will need.

**Comment by jdenen** [comment]
The default in Azure is to NOT make db/username the same, unlike AWS. I consider this a bug that we're forcing others into a configuration that goes against a cloud vendor's grain.

**Comment by jeffgrunewald** [comment]
I'm misinterpreting the "POSTGRES_NAME" then; I assume you mean that variable is re-used passing in both the DB name _and_ the user name. I would honestly prefer that be renamed more explicitly as part of this implementation to "POSTGRES_USERNAME" and "POSTGRES_DBNAME" or similar.

**Comment by jdenen** [comment]
Yeah, that's exactly what's going on. `POSTGRES_NAME` is used for both db and user names.

`POSTGRES_USERNAME` and `POSTGRES_DBNAME` works for me.

**Comment by jeffgrunewald** [comment]
Naming is hard.
